### PR TITLE
add: `onlyoffice-desktopeditors-deb`

### PR DESCRIPTION
**Pull request by soumyatheman**

### CODE REVIEW

**Comment by Henryws** [comment]
Fix the title to include backticks around the name of the package please.

File: packages/onlyoffice-desktopeditors-deb/onlyoffice-desktopeditors-deb.pacscript [hunk 1]
```diff
@@ -0,0 +1,9 @@
+name="onlyoffice-desktopeditors-deb"
+gives="onlyoffice-desktopeditors"
+version="7.2.1"
+replace="${gives}"
+url="https://github.com/ONLYOFFICE/DesktopEditors/releases/download/v${version}/onlyoffice-desktopeditors_amd64.deb"
+description="ONLYOFFICE Desktop Editors is a free office suite that combines text, spreadsheet and presentation editors allowing to create, view and edit documents stored on your Linux PC without an Internet connection."
```
[the remark this answers is not in the document]
Comment (soumyatheman) [reply packages/onlyoffice-desktopeditors-deb/onlyoffice-desktopeditors-deb.pacscript:6]: Again this is from the website

[the remark this answers is not in the document]
Comment (soumyatheman) [reply packages/onlyoffice-desktopeditors-deb/onlyoffice-desktopeditors-deb.pacscript]: done it. check

[the remark this answers is not in the document]
Comment (soumyatheman) [reply packages/onlyoffice-desktopeditors-deb/onlyoffice-desktopeditors-deb.pacscript]: please resolve the change now

[the remark this answers is not in the document]
Comment (Henryws) [reply packages/onlyoffice-desktopeditors-deb/onlyoffice-desktopeditors-deb.pacscript]: I'm waiting for the CI to pass.

**Pull request closed without merging**